mesh-worker-service/src/main/java/io/functionmesh/compute/rest/api/FunctionsImpl.java

```diff
@@ -69,17 +69,32 @@ private void validateRegisterFunctionRequestParams(String tenant, String namespa
         if (functionConfig == null) {
             throw new RestException(Response.Status.BAD_REQUEST, "Function config is not provided");
         }
+        Map<String, Object> customConfig = worker().getWorkerConfig().getRuntimeCustomizerConfig();
```

Comment:
I think we should use `FunctionsWorkerServiceCustomConfigs` to get customized worker service configs instead of `RuntimeCustomizerConfig`.
```suggestion
        Map<String, Object> customConfig = worker().getWorkerConfig().getFunctionsWorkerServiceCustomConfigs();
```